[2024-01-18T22:11:32.844+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Google_X_mention_score.test_model_exp scheduled__2024-01-01T04:30:00+00:00 [queued]>
[2024-01-18T22:11:32.865+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Google_X_mention_score.test_model_exp scheduled__2024-01-01T04:30:00+00:00 [queued]>
[2024-01-18T22:11:32.866+0300] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-01-18T22:11:32.868+0300] {taskinstance.py:1289} INFO - Starting attempt 1 of 2
[2024-01-18T22:11:32.869+0300] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-01-18T22:11:32.925+0300] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): test_model_exp> on 2024-01-01 04:30:00+00:00
[2024-01-18T22:11:32.957+0300] {standard_task_runner.py:55} INFO - Started process 6166 to run task
[2024-01-18T22:11:32.982+0300] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'Google_X_mention_score', 'test_model_exp', 'scheduled__2024-01-01T04:30:00+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/google_mention_score.py', '--cfg-path', '/tmp/tmpt6ftd17s']
[2024-01-18T22:11:32.994+0300] {standard_task_runner.py:83} INFO - Job 51: Subtask test_model_exp
[2024-01-18T22:11:33.195+0300] {task_command.py:389} INFO - Running <TaskInstance: Google_X_mention_score.test_model_exp scheduled__2024-01-01T04:30:00+00:00 [running]> on host xflow
[2024-01-18T22:11:33.330+0300] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=Google_X_mention_score
AIRFLOW_CTX_TASK_ID=test_model_exp
AIRFLOW_CTX_EXECUTION_DATE=2024-01-01T04:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-01T04:30:00+00:00
[2024-01-18T22:11:33.334+0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-01-18T22:11:33.335+0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /home/angry/mlops_4/scripts/test_model.py exp']
[2024-01-18T22:11:33.383+0300] {subprocess.py:86} INFO - Output:
[2024-01-18T22:11:40.880+0300] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-18T22:11:40.934+0300] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=Google_X_mention_score, task_id=test_model_exp, execution_date=20240101T043000, start_date=20240118T191132, end_date=20240118T191140
[2024-01-18T22:11:40.999+0300] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-01-18T22:11:41.035+0300] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
